lib/Doctrine/Common/DataFixtures/Purger/PHPCRPurger.php

```diff
@@ -14,36 +14,20 @@
 class PHPCRPurger implements PurgerInterface
 {
     /**
-     * DocumentManager instance used for persistence.
-     * @var DocumentManager
+     * @var DocumentManagerInterface
```

Comment:
@dbu in which version was this interface introduced? Should we add a `conflict` clause to `composer.json`?

Reply:
it was added in 2013, was not aware that we had releases before that time :-D

its version 1.3.0, will add a conflicts